gdcdictionary/tests/utils.py

```diff
@@ -15,7 +15,7 @@
 
 def load_yaml(path):
     with open(path, 'r') as f:
-        return yaml.load(f)
+        return yaml.safe_load(f)
```

Comment:
:clap:

Reply:
For any future readers:

Even though we probably wouldn't experience malicious uses of `yaml.load()` in this case, given that we aren't running user-submitted yaml, it's still a good practice to use `safe_load()` over `load()`.

https://security.openstack.org/guidelines/dg_avoid-dangerous-input-parsing-libraries.html